src/plugins/permissionsViewer/components/icons.tsx

```diff
@@ -29,30 +35,38 @@ export function PermissionDeniedIcon() {
     );
 }
 
-export function PermissionAllowedIcon() {
+export function PermissionDefaultIcon(): JSX.Element {
```

Comment:
```suggestion
export function PermissionDefaultIcon() {
```
don't type the return type unless necessary (eg overloads or recursive funcs)